digits/extensions/data/imageProcessing/data.py

```diff
@@ -0,0 +1,154 @@
+# Copyright (c) 2016, NVIDIA CORPORATION.  All rights reserved.
+from __future__ import absolute_import
+
+import math
+import numpy as np
+import os
+import random
+
+from digits.utils import image, subclass, override, constants
+from ..interface import DataIngestionInterface
+from .forms import DatasetForm
+
+TEMPLATE = "template.html"
+
+
+@subclass
+class DataIngestion(DataIngestionInterface):
+    """
+    A data ingestion extension for an image processing dataset
+    """
+
+    def __init__(self, **kwargs):
+        """
+        Init
+        """
+        # the parent __init__ method automatically populates this
```

Comment:
This docstring is pretty useless. Could you roll the comment up into the docstring?